annotations.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	mkr "github.com/mackerelio/mackerel-client-go"
+	"github.com/mackerelio/mkr/logger"
+	"gopkg.in/urfave/cli.v1"
+	"os"
+)
+
+var commandAnnotations = cli.Command{
+	Name: "annotations",
+	Subcommands: []cli.Command{
+		{
+			Name:        "create",
+			Usage:       "create annotation",
+			Description: "Creates an annotation.",
+			Action:      doAnnotationsCreate,
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "title", Value: "", Usage: "Title for annotation."},
+				cli.StringFlag{Name: "description", Value: "", Usage: "Description for annotation."},
+				cli.IntFlag{Name: "from"},
+				cli.IntFlag{Name: "to"},
+				cli.StringFlag{Name: "service, s", Value: "", Usage: "Service name for annotation."},
+				cli.StringSliceFlag{
+					Name:  "role, r",
+					Value: &cli.StringSlice{},
+					Usage: "Roles for annotation. Multiple choices are allowed.",
+				},
+			},
+		},
+		{
+			Name:        "list",
+			Usage:       "list annotations",
+			Description: "Shows annotations by service name and duration(from and to).",
+			Action:      doAnnotationsList,
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "service, s", Value: "", Usage: "Service name for annotation."},
+				cli.IntFlag{Name: "from"},
+				cli.IntFlag{Name: "to"},
+			},
+		},
+		{
+			Name:        "update",
+			Usage:       "update annotation",
+			Description: "Updates an annotation.",
+			Action:      doAnnotationsUpdate,
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "id", Value: "", Usage: "Annotation ID."},
+				cli.StringFlag{Name: "service, s", Value: "", Usage: "Service name for annotation."},
+				cli.StringFlag{Name: "title", Value: "", Usage: "Title for annotation."},
+				cli.StringFlag{Name: "description", Value: "", Usage: "Description for annotation."},
+				cli.IntFlag{Name: "from"},
+				cli.IntFlag{Name: "to"},
+				cli.StringSliceFlag{
+					Name:  "role, r",
+					Value: &cli.StringSlice{},
+					Usage: "Roles for annotation. Multiple choices are allowed.",
+				},
+			},
+		},
+		{
+			Name:        "delete",
+			Usage:       "delete annotation",
+			Description: "Delete graph annotation by annotation id.",
+			Action:      doAnnotationsDelete,
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "id", Value: "", Usage: "Reason of closing alert."},
+			},
+		},
+	},
+}
+
+func doAnnotationsCreate(c *cli.Context) error {
```

Comment:
I sent a pull request to return a created graph annotation (https://github.com/mackerelio/mackerel-client-go/pull/41), and I modified `doAnnotationsCreate` to print a created graph annotation.